internal/services/engines/nginx/rules.go

```diff
@@ -32,9 +36,73 @@ func extensions() []string {
 func rules() []engine.Rule {
 	return []engine.Rule{
 		// Not rules
-		not.NewNginxNotIncludeXFrameOptionsHeader(),
-		not.NewNginxNotIncludeXContentTypeOptionsHeader(),
-		not.NewNginxNotIncludeContentSecurityPolicyHeader(),
-		not.NewNginxNotIncludeServerTokensOff(),
+		NewIncludeXFrameOptionsHeader(),
+		NewIncludeXContentTypeOptionsHeader(),
+		NewIncludeContentSecurityPolicyHeader(),
+		NewIncludeServerTokensOff(),
+	}
+}
+
+func NewIncludeXFrameOptionsHeader() text.TextRule {
```

Comment:
Why not divide like the others? `rules.go` and `role_manager.go`

Reply:
I think it's because it has fewer rules than others. Do you think this can be confusing?

Reply:
As everyone tends to grow, in the future we would have to make these changes anyway.

Reply:
Make sense, I gonna split these rules too.